django/db/models/fields/mixins.py

```diff
@@ -12,6 +12,22 @@ def get_cached_value(self, instance, default=NOT_PROVIDED):
         try:
             return instance._state.fields_cache[cache_name]
         except KeyError:
+            # An ancestor link will exist if this field is defined on a MTI
```

Comment:
Maybe should use "multi-table inheritance" instead of MTI as I noticed the acronym isn't used much.